src/controllers/mainController.ts

```diff
@@ -1178,6 +1187,17 @@ export default class MainController implements vscode.Disposable {
 
 			await this.sanitizeConnectionProfiles();
 
+			if (e.affectsConfiguration(Constants.cmdObjectExplorerGroupBySchemaFlagName)) {
+				(await this._objectExplorerProvider.getChildren()).forEach((n: TreeNodeInfo) => {
+					try {
+						this._objectExplorerProvider.refreshNode(n);
+					} catch (e) {
+						Utils.logDebug(e.toString());
+						Utils.showErrorMsg(e.toString());
```

Comment:
Currently if something causes a couple of the refreshes to fail then this will get really spammy. Two options I would suggest you consider : 

1. Just logging to console
2. Only display a single message to the user if any of the node expansions failed - and optionally direct them to the console/output window logs for further information

Reply:
Don't errors get automatically logged to console?